Changed Home page title to explorer by modular cloud

Aswanth-c opened the pull request:
Fixed the mentioned Issue  https://github.com/modularcloud/explorer/issues/62

lzrscg reviewed (changes requested):
This won't work, unfortunately. What needs to happen is we need to edit the whitelabel object (from `getWhitelabel()`) to include a custom whitelabel field for each explorer.

Thanks for getting this started :)

lzrscg commented:
for example, celestiascan.com should say "CelestiaScan by Modular Cloud"

Aswanth-c commented:
> This won't work, unfortunately. What needs to happen is we need to edit the whitelabel object (from `getWhitelabel()`) to include a custom whitelabel field for each explorer.
> 
> Thanks for getting this started :)

Hey Lz, sorry for the misunderstanding.

I have fixed the issue by
creating a property  Inside the whiteLableObject variable and naming it "title". 
In most of the cases, I  combined the name array and added those values to the title property. 
except In world and default env where the names  are ["modular" "cloud"], I changed it to explorer

lzrscg commented:
thanks @Aswanth-c this is great! I was thinking maybe the full title could go in the metadata. So for instance,

"Explorer by Modular Cloud"
"CelestiaScan by Modular Cloud"
"Something else entirely"

Could all be the title.

Perhaps something like this could be added to the whitelabel config:
```
{
   "metadata": {
      "title": "Explorer by Modular Cloud",
      "description": "A modular block explorer built by Modular Cloud",
   }
}
```

Aswanth-c commented:
> erhaps something like this could be added to the whitelabel config:

Done!

lzrscg commented on apps/web/lib/utils.ts:
```diff
@@ -43,7 +43,10 @@ export async function fetchLoad(props: FetchLoadArgs) {
   }
 }
-
+export type metaData = {
```
sorry to be a nitpicker 🥺👉👈 but can you rename this "Metadata"

Aswanth-c replied:
Haha nice catch
Fixed!

lzrscg reviewed (changes requested):
LGTM after the capitalization fix :D

lzrscg commented:
i fixed this in a previous update already, thanks for putting this together though!
